WPMedia/fusion-news-theme-blocks
# Themes 1986: Add Alt tags to images where missing

## Description

#### Jira Ticket: [THEMES-1986](https://arcpublishing.atlassian.net/browse/THEMES-1986)

Adding alt tags to images where they were not previously (as necessary)

## Test Steps

1. Checkout this branch `git checkout themes-1986`
2. Run fusion repo with linked blocks `npx fusion start -f -l`
3. Verify that the medium and small top table list promos have alt values: http://localhost/pagebuilder/editor/curate?p=puOvG1XS6AaSAy1u&v=vy3r49MKAaSAy1u
4. Verify that the alt property is applied to the search-results-list, results-list, numbered-list, and simple list where the header is NOT enabled in the block config: http://localhost/pagebuilder/editor/curate?p=pldHYSIJj7fEkicu&v=vOlaaYHh67fEkicu


[THEMES-1986]: https://arcpublishing.atlassian.net/browse/THEMES-1986?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
